backend/scripts/update_skill_binary.py

```
@@ -87,12 +90,17 @@ def update_skill_binary(db: Session, skill_name: str, skill_folder: Path) -> boo
     if skill_binary:
         skill_binary.binary_data = zip_data
         skill_binary.file_size = len(zip_data)
+        skill_binary.file_hash = zip_hash
+        skill_binary.file_name = zip_name
+        skill_binary.type = None
         print(f"Updated existing SkillBinary record")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)